install_config/certificate_customization.adoc

```diff
@@ -108,20 +108,22 @@ servingInfo:
   bindNetwork: tcp4
   certFile: master.server.crt <1>
   clientCA: ""
-  keyFile: master.server.key <1>
+  keyFile: master.server.key <2>
   maxRequestsInFlight: 0
   requestTimeoutSeconds: 0
   namedCertificates:
-    - certFile: wildcard.example.com.crt <2>
-      keyFile: wildcard.example.com.key <2>
+    - certFile: wildcard.example.com.crt <3>
+      keyFile: wildcard.example.com.key <4>
       names:
         - "openshift.example.com"
   metricsPublicURL: "https://metrics.os.example.com/hawkular/metrics"
 
 ----
 
-<1> Path to certificate and key files for the CLI and other API calls.
-<2> Path to certificate and key files for the web console.
+<1> Path to the certificate file for the CLI and other API calls.
+<2> Path to the key file for the CLI and other API calls.
+<3> Path to the certificate file for the web console. If necessary, concatenate all of the required files that form your xref:../install_config/certificate_customization.adoc#using-certificate-chains[certificate chain] for the certificate file that is provided to the `certFile` parameter.
```

Comment:
I'm not sure `for the web console` is accurate enough here. I noticed we have such description about the named cert in the [overview](http://file.rdu.redhat.com/~ahoffer/2020/BZ-1691019/install_config/certificate_customization.html#overview):
```
Administrators can configure custom serving certificates for the public host names of the OpenShift Container Platform API and web console.
```